torchscript_export.py

```diff
@@ -0,0 +1,126 @@
+import os
+import sys
+import argparse
+from pathlib import Path
+
+import torch
+
+# Add 'core', which contains the imports
+# for MonSter to the module search path.
+sys.path.append("core")
+
+from monster import Monster
+from utils.utils import InputPadder
+
+
+class ScriptableMonSter(torch.nn.Module):
+    def __init__(self, monster: torch.nn.Module):
+        super().__init__()
+        self.monster = monster
+
+    def forward(
+        self,
+        left_image: torch.Tensor,
+        right_image: torch.Tensor
+    ) -> torch.Tensor:
+
+        left_image = left_image.unsqueeze(0)
+        right_image = right_image.unsqueeze(0)
+
+        # Now pad to make divisible by 32
+        padder = InputPadder(left_image.shape, divis_by=32)
+        left_image, right_image = padder.pad(inputs=[left_image, right_image])
+
+        # Run the monster model
+        disp = self.monster(left_image, right_image, iters=24, test_mode=True)
+
+        # Remove padding
+        disp = padder.unpad(disp)
+
+        disp = torch.squeeze(disp[:, 0, :, :])
+
+        return disp
+
+
+def parse_args():
```

Comment:
Please follow an approach similar to the one we are already following for argument parsing: https://github.com/fizyr-private/torch-train-evaluate/blob/8f00a1bf5e0112e6333c75d2c3f33d69cd293532/torch_train_evaluate/bin/to_script.py#L30